data/mods/gen4/moves.ts

```diff
@@ -155,6 +155,12 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		inherit: true,
 		recoil: [1, 3],
 	},
+	brickbreak: {
+		inherit: true,
+		onTryMove(_attacker, defender, _move) {
+			defender.side.removeSideCondition('reflect');
+		},
```

Comment:
```suggestion
		onTryMove(attacker, defender) {
			defender.side.removeSideCondition('reflect');
		},
```
Usually, PS either omits the names of unused arguments (if they're after the ones that are used) or keep them as-is with no underscore.

Reply:
Fixed, thank you!